### ISSUE TITLE
Setup for module imports

### ISSUE THREAD

**Issue by NickiRom** [victorwyee/lungbox#4]
Hi Victor,  could you comment on how you expect users to install your package? train.py requires installation of the libs module for example, but there is no setup.py to install lungbox and pip install git+... is not working

**Comment by victorwyee** [comment]
Yup! I'm putting a working requirements.txt and setup.py together now.

**Comment by victorwyee** [comment]
@NickiRom -- I've added and (manually) tested

```
pip install -r requirements.txt
python setup.py install
```

To train, the command is currently
```
python src/notebooks/model1_maskrcnn/train.py \
    --subset_size=1000 \
    --validation_split=0.2 \
    --epochs=1
```
But, all of my data is currently on S3, and all data loading functions are currently designed to hit S3. I can add your AWS email to the bucket. I'm still getting to putting sample data into this repo, and local data loading functions. Command line inference is still in progress.
